src/scss/components/optional/_cuisines-widget.scss

```diff
@@ -0,0 +1,127 @@
+/**
+* Components > Cuisines Widget
+* =================================
+*
+* Example: Homepage – Popular cuisines.
+*
+* The `c-cuisinesWidget` component is an optional mixin within Fozzie.
+* If you'd like to use it in your project you can include it by adding `@include cuisinesWidget();` into your SCSS dependencies file.
+*
+* Documentation:
+* https://fozzie.just-eat.com/styleguide/ui-components/cuisines-widget
+*/
+
+$cuisinesWidget-titleColour: $white;
+$cuisinesWidget-defaultBackground: $brand--red;
+$cuisinesWidgetGrid-breakpoint-narrow: 570px;
+$cuisinesWidgetGrid-breakpoint-wide: 1241px;
+$cuisinesWidget-boxShadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
+$cuisinesWidget-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0 rgba(0, 0, 0, 0.04);
+$cuisinesWidget-gradient: linear-gradient(to top, rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0));
+
+@mixin cuisinesWidget() {
+
+   .c-cuisinesWidget {
+       display: block;
+       border-radius: 8px;
+       overflow: hidden;
+       flex: 0 0 auto;
+       position: relative;
+       background-color: $cuisinesWidget-defaultBackground;
+       box-shadow: $cuisinesWidget-boxShadow;
+
+       &:hover {
+           box-shadow: $cuisinesWidget-hoverBoxShadow;
+       }
+
+       .c-cuisinesWidget-name {
```

Comment:
Shouldn't need to nest this selector inside `.c-cuisinesWidget`.  Just makes the specificity higher otherwise (same for all child classes below)

Reply:
✅